<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>controlpanel</class>
 <widget class="QWidget" name="controlpanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>811</width>
     <height>221</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="joints">
    <property name="toolTip">
     <string>Joints
</string>
    </property>
    <attribute name="title">
     <string>Joints</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>16</y>
       <width>161</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="baseButton">
        <property name="text">
         <string>Base</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="shoulderButton">
        <property name="text">
         <string>Shoulder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="elbowButton">
        <property name="text">
         <string>Elbow</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="wrist1Button">
        <property name="text">
         <string>Wrist 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="wrist2Button">
        <property name="text">
         <string>Wrist 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="wrist3Button">
        <property name="text">
         <string>Wrist 3</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>391</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="turnRightButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="turnLeftButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="manipulator">
    <attribute name="title">
     <string>Manipulator</string>
    </attribute>
    <widget class="QPushButton" name="manipulatorUpButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>0</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="manipulatorLeftButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="manipulatorRightButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="manipulatorDownButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="turnManipulatorLeft">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>27</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="turnManipulatorRight">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>27</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>151</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="angleA">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>α</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="angelB">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>β </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="angleBtext"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="angleC">
          <property name="text">
           <string>γ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="angleCtext">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="angleD">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>δ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="angelDtext"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <zorder>manipulatorLeftButton</zorder>
    <zorder>manipulatorRightButton</zorder>
    <zorder>manipulatorDownButton</zorder>
    <zorder>turnManipulatorLeft</zorder>
    <zorder>turnManipulatorRight</zorder>
    <zorder>layoutWidget_3</zorder>
    <zorder>manipulatorUpButton</zorder>
   </widget>
   <widget class="QWidget" name="state">
    <attribute name="title">
     <string>States</string>
    </attribute>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>81</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="joint1Label">
          <property name="text">
           <string>Joint 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="joint1Pixmap">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="joint3Label">
          <property name="text">
           <string>Joint3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="joint3Pixmap">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="joint2Label">
          <property name="text">
           <string>Joint 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="joint2Pixmap">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>81</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="shoulderLabel">
          <property name="text">
           <string>Shoulder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="shoulderPixmap">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="baseLabel">
          <property name="text">
           <string>Base</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="basePixmap">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="elbowLabel">
          <property name="text">
           <string>Elbow</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="elbowPixmap">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>0</y>
     <width>181</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="groupVerticalLay"/>
   </widget>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
